lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestFailedTaskEventHandler.cs

```diff
@@ -91,6 +92,11 @@ public void OnNext(IAllocatedEvaluator value)
             public void OnNext(IFailedTask value)
             {
                 Logger.Log(Level.Error, FailedTaskMessage);
+                if (value.Message == null || value.Message != ExpectedExceptionMessage)
```

Comment:
Only differences are that `.Equals` throws NPE if compared to `null`, and `==` is called based on compile time types while `.Equals` is called based on runtime types, but in this case it doesn't really matter.